example/generic_social_network/app/resources/user_read_history_resource.py

```diff
@@ -17,9 +17,13 @@ class UserReadHistoryResource(APIResource):
 class UserReadHistorySchema(Schema):
     SCHEMA = {
         'type': 'user-read-history',
-        'id': StringAttribute(serializer_method='user_read_history_id'),
+        'id': StringAttribute()
+                .read_from(serializer_method='user_read_history_id')
+                .self_explanatory(),
```

Comment:
This isn't very optimal. I'm guessing this will be fixed when I move toward functional schemas (`get_attributes()`, `get_id_attribute()` etc).

For now, `id` will have to be `self_explanatory()` to pass configuration verification